ios-involves/ios-involves/Application/AppDelegate.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  AppDelegate.swift
+//  ios-involves
+//
+//  Created by Danilo Pena on 20/02/20.
+//  Copyright © 2020 Danilo Pena. All rights reserved.
+//
+
+import UIKit
+import TraktKit
+
+extension Notification.Name {
+    static let TraktSignedIn = Notification.Name(rawValue: "TraktSignedIn")
+}
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    private struct Constants {
```

Comment:
```suggestion
    private enum Constants {
```
Eu particulamente gosto de criar Constants com enum.
Aqui está o motivo:
`The advantage of using a case-less enumeration is that it can't accidentally be instantiated and works as a pure namespace.`
https://stackoverflow.com/questions/38585344/swift-constants-struct-or-enum